.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "jest.jestCommandLine": "jest --detectOpenHandles"
+}
```

Comment:
Missing EOL, but this file should not be here in the first place. Probably want to put `.vscode` in your global `.gitignore`.

Reply:
Ah yes, that sneaked in there 👀